README.md

```diff
@@ -8,7 +8,7 @@ This guide provides an introduction to develop a [JavaScript](https://developer.
 > The best one sentence to describe an SDK is: _"The SDK is the connection bridging the gap between users and the (browser) machine."_
 
 
-At the end of this guide, the SDK will be able to run in browsers, desktop, mobile web and various other platforms capable of running JavaScript. 
+By using this guide, the SDK will be able to run in browsers, desktop, mobile web and various other platforms capable of running JavaScript. 
```

Comment:
trying ot get this sent to hueitan...  the previous changes were committed into this forked branch,    you can check the changes and merge with your main branch... hope it helps..